charts/kcp-operator/README.md

```diff
@@ -1,6 +1,33 @@
 # KCP Helm Chart - kcp-operator
 
 This Helm chart deploys an instance of the [kcp-operator](https://github.com/kcp-dev/kcp-operator).
-For more documentation, check out [docs.kcp.io/kcp-operator](https://docs.kcp.io/kcp-operator).
+
+## Requirements
+
+* [cert-manager](https://cert-manager.io/docs/installation)
+
+## Usage
+
+[Helm](https://helm.sh) must be installed to use the charts. Please refer to Helm's
+[documentation](https://helm.sh/docs) to get started.
+
+Once Helm has been set up correctly, add the repo as follows:
+
+    helm repo add kcp https://kcp-dev.github.io/helm-charts
+
+If you had already added this repo earlier, run `helm repo update` to retrieve the latest versions
+of the packages. You can then run `helm search repo kcp` to see the charts.
```

Comment:
kcp-operator

Reply:
No, this is correct here. It searches the "kcp" repo.